strapi/strapi
# [Custom fields] Create custom fields plugin and first Strapi custom field: Color Picker

### What does it do?

- Adds a new package to the monorepo `@strapi/custom-fields` 
- Creates a new custom field: Color Picker

### Why is it needed?

Color picker is the most requested custom field

### How to test it?

- Run `yarn setup`, then run the `examples/getstarted` app with `yarn develop --watch-admin`
- Select any content-type in the CTB, add a new field, select custom, select Color Picker (not Color!), save the content-type
- Head to your content-type in the content manager, create a new entity
- You should see an input for the custom field that matches the design system
- It should have a preview circle of the color and a hex value, it should default to white #FFFFFF
- It should show a ui when clicking the input
- It should update the hex value in the UI and on the toggle button preview
- You should be able to update the HEX on the UI's input and the change should show in the color saturation / hue 
- It should close when clicking outside the UI or when clicking the toggle button
- The entity should save to the db
